tests/components/homekit_controller/test_sensor.py

```diff
@@ -105,6 +116,8 @@ def create_battery_level_sensor(accessory):
     state = await helper.poll_and_get_state()
     assert state.state == "20"
 
+    assert state.attributes["device_class"] == DEVICE_CLASS_ILLUMINANCE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5069901)